debian/rules

```diff
@@ -5,8 +5,17 @@ PACKAGE = $(shell dh_listpackages|head -n 1|cut -d '-' -f 1)
 SYSCONFDIR = "/etc"
 DESTDIR = "debian/tmp"
 
+ifeq ($(shell lsb_release -sr), 14.04)
+        SUBSTVARS = -Vjre:Depends="openjdk-7-jre-headless" -Vjdk:Depends="openjdk-7-jdk" -Vinit:"Depends:jsvc"
+else
+        SUBSTVARS = -Vjre:Depends="openjdk-8-jre-headless" -Vjdk:Depends="openjdk-8-jdk" -Vinit:"Depends:systemd"
```

Comment:
@wido should this be written like:   `-Vinit:Depends="systemd"`. Would it be possible to support both systemd/initd environments by adding the dependency like `systemd | jsvc` -- this way it would try to install both? On (systemd based) Ubuntu 16.04 systemd will be already installed, we can install jsvc for all systems -- so why not just depend on jsvc alone? On 14.04, the pkgs can continue to use jsvc/initd script and on 16.04 it can uses systemd, both using the same packages.

I want us to avoid hosting two separate deb repositories (one for non-systemd based and one for systemd based) if possible. Thoughts?